database/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Copyright 2018 Bitwise IO
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------------
+
+# to build:
+# docker build -f ./subscriber/Dockerfile -t sawtooth-bond-databse .
```

Comment:
This comments are bad - because they are always wrong. In this case, I think this is a cut-n-paste error from subscriber/Dockerfile and there is a possible typo in 'sawtooth-bond-databse'.

My recommendation is to consolidate build instructions to a top-level file that can be tested all at once (these individual file comments never get tested).